### PR TITLE
nullable slug must not be null if base field has "0"

### PR DESCRIPTION
**Pull request by vetalt**
nullable slug must not be null if base field has "0"

### CODE REVIEW

File: lib/Gedmo/Sluggable/SluggableListener.php [hunk 1]
```diff
@@ -381,7 +381,7 @@ private function generateSlug(SluggableAdapter $ea, $object)
                     $slug = substr($slug, 0, $mapping['length']);
                 }
 
-                if (isset($mapping['nullable']) && $mapping['nullable'] && !$slug) {
+                if (isset($mapping['nullable']) && $mapping['nullable'] && strlen($slug) === 0) {
```
Comment (l3pp4rd) [inline lib/Gedmo/Sluggable/SluggableListener.php]: ah yes, good old PHP tricks ;)

**Comment by l3pp4rd** [comment]
thanks ;)

**Comment by umpirsky** [comment]
@l3pp4rd Can you tag maintenance release including this fix please?

**Comment by l3pp4rd** [comment]
should be available ;)

**Pull request merged**